File: src/basic.py
from dotenv import load_dotenv
load_dotenv()

from langchain_google_genai import ChatGoogleGenerativeAI
llm = ChatGoogleGenerativeAI(model="gemini-2.0-flash-lite")

result = llm.invoke("LLMについて教えて下さい")
print(result.content)

File: js/basic.js
import { config } from "dotenv";
config();

import { ChatGoogleGenerativeAI } from "@langchain/google-genai";

const llm = new ChatGoogleGenerativeAI({ model: "gemini-1.5-flash" });

const result = await llm.invoke("LLMについて教えて下さい");

console.log(result.content);
